### PR TITLE
[9.0] update upstream 75388edd9c

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Merging upstream Odoo up until commit https://github.com/odoo/odoo/commit/75388edd9c6cb54ee04c2bcac6886dbedd717700.

There were no conflicts.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: openerp/addons/base/migrations/9.0.1.3/noupdate_changes.xml [hunk 1]
```diff
@@ -145,6 +145,7 @@
       <field name="active" eval="False"/>
     </record>
     <record id="res_partner_title_doctor" model="res.partner.title">
+      <field name="domain"/>
```
Comment (pedrobaeza) [inline openerp/addons/base/migrations/9.0.1.3/noupdate_changes.xml]: This field no longer exists. That's why it appears in the noupdate. Please remove it from all for avoiding the warnings.

Comment (MiquelRForgeFlow) [reply openerp/addons/base/migrations/9.0.1.3/noupdate_changes.xml]: Done

File: openerp/addons/base/migrations/9.0.1.3/noupdate_changes.xml [hunk 2]
```diff
@@ -607,6 +608,7 @@
       <field eval="27" name="phone_code"/>
     </record>
     <record id="GTQ" model="res.currency">
+      <field name="company_id"/>
```
Comment (pedrobaeza) [inline openerp/addons/base/migrations/9.0.1.3/noupdate_changes.xml]: This one also doesn't exist. You should check WARNING in logs for knowing it.

**Comment by MiquelRForgeFlow** [comment]
I thought to also include here https://github.com/OCA/OpenUpgrade/pull/1449 :)

**Comment by MiquelRForgeFlow** [comment]
I don't understand, really. I created locally a v8 database with demo data. I installed all `l10n_*` modules (except `l10n_be_hr_payroll_account` that for an unknown reason I couldn't install it). Then I migrated it and this error didn't appear.

**Comment by pedrobaeza** [comment]
@mreficent v8 database is static and was generated by @StefanRijnhart some time ago. For sure this is a new constraint that the old DB don't comply in demo data, so you should remove that and do it via SQL code.

**Comment by MiquelRForgeFlow** [comment]
Ok, ready to merge. I just removed it. I won't waste more time in something that I really don't bother  😥

**Comment by pedrobaeza** [comment]
Well, can you remind me which one was, because maybe it's important (I think it was the receivable account type, which should be reconciled or reconcile system won't work).

**Comment by MiquelRForgeFlow** [comment]
Both the receivable and the payable. Both have to be related to reconcilable accounts.

**Comment by pedrobaeza** [comment]
Then the SQL code is needed. What I don't get is why this was working before and not now...

**Comment by MiquelRForgeFlow** [comment]
Could you merge this PR and do that SQL code in another PR? I ask it because this PR is unrelated with that thing of the accounts.

**Comment by pedrobaeza** [comment]
OK, but I'm not going to work on that part, as I don't need it for now in my projects.

**Pull request merged**